electron-builder.yml

```
@@ -30,6 +30,9 @@ extraResources:
   - from: ./runtime/
     to: app.asar.unpacked/runtime
     filter: ['**/*']
+  - from: ./app.asar.unpacked/internal-proxy
+    to: app.asar.unpacked/internal-proxy  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)